.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: 'airbnb-base',
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  rules: {
+    'prefer-default-export': 'off',
+    'import/prefer-default-export': 'warn',
+    'no-plusplus': 'off',
+    'arrow-parens': 'warn',
```

Comment:
Хочу чтобы переопределения правил были более специфичными в требованиях к коду, чтобы было больше единообразия, а так же чтобы видеть что вы реально разбирались со значением некоторых правил. В данном случае более правильным было бы следующее
```suggestion
    'arrow-parens': ['warn', 'as-needed'],
```